web/components/PostCreationModal/Form/FormSlide.tsx

```diff
@@ -0,0 +1,229 @@
+import {
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Select,
+  Stack,
+  useDisclosure,
+  Text,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useFormik } from "formik";
```

Comment:
Remove all unused imports (also remove any unused libraries you may have installed that you aren't using now)